http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/api/AppV2Routes.scala

```diff
@@ -204,7 +206,9 @@ object AppV2Routes {
                                dp,
                                ea.getOrElse(List.empty),
                                wsi,
-                               swi
+                               swi,
+                               adtm,
+                               adte
```

Comment:
```suggestion
                               at,
                               ae
```
... this should be modified based on how/what you choose above.